msg/messages.js

```diff
@@ -8,8 +8,8 @@
  * @fileoverview English strings.
  * @author fraser@google.com (Neil Fraser)
  *
- * After modifying this file, either run "build.py" from the parent directory,
- * or run (from this directory):
+ * After modifying this file, either run "npm run build:langfiles" from the
+ * parent directory, or run (from this directory):
```

Comment:
I'm about to patch this same comment in another PR.

Reply:
If you were going to hit these lines in your PR on build stuff, I can just close this and wait.